Python_Engine/Query/Import.cs

```diff
@@ -47,7 +47,7 @@ public static PyObject Import(string moduleName)
                 if (confirmResult == DialogResult.Yes)
                 {
                     Output<bool, List<string>> result = Compute.InstallPythonToolkit(true);
-                    Reflection.Compute.RecordWarning("Python has been installed, please restart the UI");
+                    Reflection.Compute.RecordWarning("Python has been installed, please restart the UI or recompute the script if using Grasshopper.");
```

Comment:
Change to `or recompute the script` and leave out the reference to the specific UI.